test/uk/gov/hmrc/transitmovements/controllers/MovementsControllerSpec.scala

```diff
@@ -37,6 +37,8 @@ import org.scalatest.matchers.must.Matchers
 import org.scalatestplus.scalacheck.ScalaCheckDrivenPropertyChecks
 import play.api.http.HeaderNames
 import play.api.http.MimeTypes
+import uk.gov.hmrc.objectstore.client.ObjectSummaryWithMd5
+import uk.gov.hmrc.objectstore.client.Path
```

Comment:
nitpick: make imports alphabetical please (except for scala/java imports, which go below in their own separate block)